.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules/
+package.json
+package-lock.json
+test.md
```

Comment:
- [ ] [Optional] [Highly recommended] Nice job adding the `node_modules` directory and `test.md` file to `.gitignore` but please in the future don't add `package.json` to `gitignore `because it has the info about the libraries and dependencies you used in your project and anyone needs to test it have to install those packages.

Reply:
Thanks Omar, I was doubting about this. Should I add this files to the repository for future projects? I was looking the linter repositories as an example of what files are included.

Reply:
@Omar-Muhamad

Reply:
Don't worry, Kindly include package.json file to your next projects and don't add it to .gitignore file again.
Have a great day 😊